### PR TITLE
Rebase my own changes into EnZa's draft

### PR DESCRIPTION
**Pull request by Kichura**
Enhances the rest of the missing details i have made on my own, Do note that this should be previewed before merging into branch.

### CODE REVIEW

**Comment by FlorianMichael** [comment]
I'll wait for Jo0001 to review it

File: src/main/resources/config.json [hunk 1]
```diff
@@ -82,9 +82,9 @@
     {
       "command": "more",
       "messages": {
-        "plugin": "Please provide more information:\n- Enable `debug=true` in server.properties and recreate the issue.\n- Provide `/viaversion dump` link, this will allow us to work out incompatibilities\n- Ensure you've said the **client-version** the issue happens with.",
-        "mod": "Please provide more information:\n- Provide `/viafabricplus dump` link, this will allow us to work out incompatibilities\n- Provide your `latest.log` file\n- Ensure you've said the **target-version** the issue happens with.",
-        "proxy": "Please provide more information:\n- Go to `Advanced` and run `Create ViaVersion dump` and provide us the link.\n- Go to `Advanced` and run `Upload latest log file` and provide us the link.\n- Ensure you've said the **Server Version** the issue happens with."
```
Comment (FlorianMichael) [inline src/main/resources/config.json:87]: Revert the proxy change

**Review by FlorianMichael — CHANGES_REQUESTED** [review]

**Review by FlorianMichael — APPROVED** [review]
Thanks

File: src/main/resources/config.json (hunk 1 above)
Comment (Kichura) [reply src/main/resources/config.json:87]: solved.

File: src/main/resources/config.json [hunk 2]
```diff
@@ -57,19 +57,19 @@
...
     },
     {
       "commands": ["blockprotocols", "blockversions", "block"],
-      "message": "You can use https://via.krusic22.com/ to generate the list for `block-protocols` in the ViaVersion config, this will allow you to allow / deny specific versions.\nYou can also use `block-versions` to specify blocked versions by using its minecraft name (\"1.17\") and operands like `< >` (you can also use https://viaversion.com/setup to generate the list). For example  `\"<1.17\"` will only allow 1.17.x and above to join"
+      "message": "You can use https://via.krusic22.com/ to generate the list for `block-protocols` in the ViaVersion config, this will allow you to allow / deny specific versions.\nYou can also use `block-versions` to specify blocked versions by using it's Minecraft name (\"1.17\") and operands like `< >`. For example `\"<1.17\"` will only allow 1.17.x and above to join.\n*(You can also use https://viaversion.com/setup to generate the list)*"
```
Comment (kennytv) [inline src/main/resources/config.json:72]: its was correct

File: src/main/resources/config.json [hunk 3]
```diff
@@ -57,19 +57,19 @@
     },
     {
       "command": "packets",
-      "message": "If you were kicked for \"You are sending too many packets!\" change `max-pps` in the ViaVersion config.yml to either a higher number or -1 to disable it. Use `/viaversion pps` in-game to view how many packets players are sending."
+      "message": "If you were kicked for \"You are sending too many packets!\", Change `max-pps` in the ViaVersion `config.yml` to either a higher number or -1 to disable it. Use `/viaversion pps` in-game to view how many packets players are sending."
```
Comment (kennytv) [inline src/main/resources/config.json:60]: lower case c

File: src/main/resources/config.json [hunk 4]
```diff
@@ -141,19 +149,19 @@
     },
     {
       "command": "error",
-      "message": "**That issue is not from Via, it's from the plugin shown in the stack trace. Try the latest version of that plugin or contact the author of it and tell them to update the Via API usage**"
+      "message": "**That issue is not from Via, it's from the plugin shown in the stack trace. Try the latest version of that plugin or contact the author of it and tell them to update the Via API usage.**"
     },
     {
       "command": "btlp",
-      "message": "There is a current issue with BungeeTabListPlus and the latest Minecraft update, we suggest following their discord for updates."
+      "message": "There is a current issue with BungeeTabListPlus and the latest Minecraft update, We suggest following their discord for updates."
```
Comment (kennytv) [inline src/main/resources/config.json:156]: back to lowercase

File: src/main/resources/config.json [hunk 5]
```diff
@@ -141,19 +149,19 @@
...
     },
     {
       "command": "reload",
-      "message": "**Do not use the `/reload` command**. It will cause issues with your server and its plugins. For more information, see https://madelinemiller.dev/blog/problem-with-reload/\n*Note: You can safely use `/minecraft:reload` to reload datapacks*"
+      "message": "**Do not use the `/reload` command**. It will cause issues with your server and it's plugins. For more information, see https://madelinemiller.dev/blog/problem-with-reload/\n*Note: You can safely use `/minecraft:reload` to reload datapacks.*"
```
Comment (kennytv) [inline src/main/resources/config.json:160]: its was correct

File: src/main/resources/config.json [hunk 6]
```diff
@@ -141,19 +149,19 @@
...
     },
     {
       "command": "height",
-      "message": "Every player on 1.7.x - 1.16.5 can still play on a  1.18+ server but  **the new world height is not supported**  (players are stuck below y=0 & only see void)\n\n*To avoid/revert this use a datapack or downgrade back to 1.17.1 or only allow 1.17+ players, see https://gist.github.com/Jo0001/4fe96cf5a78c8a00560ea985f3b9eb22 for more information*"
+      "message": "Every player on 1.7.x - 1.16.5 can still play on a 1.18+ server but **the new world height is not supported** *(players are stuck below y=0 & only see void)*\n\n*To avoid/revert this, Use a datapack or downgrade back to 1.17.1 or only allow 1.17+ players, see https://gist.github.com/Jo0001/4fe96cf5a78c8a00560ea985f3b9eb22 for more information.*"
```
Comment (kennytv) [inline src/main/resources/config.json:164]: lowercase

**Pull request merged**